include/psp2/kernel/modulemgr.h

```diff
@@ -110,21 +110,55 @@ int sceKernelStopModule(SceUID modid, SceSize args, void *argp, int flags, void
 SceUID sceKernelLoadStartModule(const char *path, SceSize args, void *argp, int flags, SceKernelLMOption *option, int *status);
 int sceKernelStopUnloadModule(SceUID modid, SceSize args, void *argp, int flags, SceKernelULMOption *option, int *status);
 
-typedef struct SceKernelFwInfo {
+typedef struct SceKernelSystemSwVersion {
 	SceSize size;
 	char versionString[0x1C];
 	SceUInt version;
 	SceUInt unk_24;
-} SceKernelFwInfo;
+} SceKernelSystemSwVersion;
+
+/* For backward compatibility */
+typedef SceKernelSystemSwVersion SceKernelFwInfo;
 
 /**
  * Gets system firmware information.
  *
- * @param[out] data - firmware information.
+ * @param[out] version - System sw version.
  *
  * @note - If you spoofed the firmware version it will return the spoofed firmware.
  */
-int sceKernelGetSystemSwVersion(SceKernelFwInfo *data);
+int sceKernelGetSystemSwVersion(SceKernelSystemSwVersion *version);
+
+// missing structs
+typedef struct SceKernelLoadModuleOpt SceKernelLoadModuleOpt;
```

Comment:
how about change the name to `SceKernelLoadModuleOption` like `SceKernelULMOption` and `SceKernelLMOption`

Reply:
I would choose `SceKernelLoadModuleOption`